Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,4 @@
-﻿{
```

Comment:
I see many tasks have no real changes at all. I presume you will exclude those from your check-in.